fog/ledger/server/src/counters.rs

```diff
@@ -1,10 +1,33 @@
 // Copyright (c) 2018-2021 MobileCoin Inc.
 
-use mc_util_metrics::{IntGauge, OpMetrics};
+use mc_util_metrics::{Histogram, IntCounter, IntGauge, OpMetrics};
 
 lazy_static::lazy_static! {
-    pub static ref OP_COUNTERS: OpMetrics = OpMetrics::new_and_registered("fog_ledger");
+          pub static ref OP_COUNTERS: OpMetrics = OpMetrics::new_and_registered("fog_ledger");
```

Comment:
can you run `cargo fmt` in this file? this code is not formatted

Reply:
I was able to make this change